smbclient.c

```diff
@@ -1345,15 +1345,16 @@ PHP_FUNCTION(smbclient_read)
 
 	if ((ZSTR_LEN(buf) = smbc_read(state->ctx, file, ZSTR_VAL(buf), count)) >= 0) {
 		RETURN_STR(buf);
+		zend_string_release(buf);
```

Comment:
no value to add this after the return statement.
Need to be after the closing }